SQL alias problem of join type.

### Which version of ShardingSphere did you use?
5.0.0-RC1-SNAPSHOT
### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-JDBC
### Expected behavior
Hope to support this feature
### Actual behavior
When the join type SQL is used and the alias of the column of the main table is omitted, the following exception will be thrown. This is different from native SQL.

Caused by: java.lang.RuntimeException: parse failed: Encountered ". date" at line 6, column 28.
Was expecting one of:
    <EOF> 
    "EXCEPT" ...
    "FETCH" ...
    "GROUP" ...
    "HAVING" ...
    "INTERSECT" ...
    "LIMIT" ...
    "OFFSET" ...
    "ORDER" ...
    "MINUS" ...
    "UNION" ...
    "WHERE" ...
    "WINDOW" ...
    "NATURAL" ...
    "JOIN" ...
    "INNER" ...
    "LEFT" ...
    "RIGHT" ...
    "FULL" ...
    "CROSS" ...
    "," ...
    "OUTER" ...
    "." <IDENTIFIER> ...
    "." <HYPHENATED_IDENTIFIER> ...
    "." <QUOTED_IDENTIFIER> ...
    "." <BACK_QUOTED_IDENTIFIER> ...
    "." <BRACKET_QUOTED_IDENTIFIER> ...
    "." <UNICODE_QUOTED_IDENTIFIER> ...
    "NOT" ...
    "IN" ...
    "<" ...
    "<=" ...
    ">" ...
    ">=" ...
    "=" ...
    "<>" ...
    "!=" ...
    "BETWEEN" ...
    "LIKE" ...
    "SIMILAR" ...
    "+" ...
    "-" ...
    "*" ...
    "/" ...
    "%" ...
    "||" ...
    "AND" ...
    "OR" ...
    "IS" ...
    "MEMBER" ...
    "SUBMULTISET" ...
    "CONTAINS" ...
    "OVERLAPS" ...
    "EQUALS" ...
    "PRECEDES" ...
    "SUCCEEDS" ...
    "IMMEDIATELY" ...
    "MULTISET" ...
    "[" ...
    "FORMAT" ...
    "." ...
    "." "*" ...
    "(" ...
	at org.apache.calcite.prepare.CalcitePrepareImpl.prepare2_(CalcitePrepareImpl.java:623) ~[calcite-core-1.26.0.jar:1.26.0]
	at org.apache.calcite.prepare.CalcitePrepareImpl.prepare_(CalcitePrepareImpl.java:508) ~[calcite-core-1.26.0.jar:1.26.0]
	at org.apache.calcite.prepare.CalcitePrepareImpl.prepareSql(CalcitePrepareImpl.java:478) ~[calcite-core-1.26.0.jar:1.26.0]
	at org.apache.calcite.jdbc.CalciteConnectionImpl.parseQuery(CalciteConnectionImpl.java:231) ~[calcite-core-1.26.0.jar:1.26.0]
	at org.apache.calcite.jdbc.CalciteConnectionImpl.prepareStatement_(CalciteConnectionImpl.java:213) ~[calcite-core-1.26.0.jar:1.26.0]
	... 128 common frames omitted

### Reason analyze (If you can)

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
Example SQL: select  s.a from user s  join  course c on s.student_id = c.stuend_id where student_name='Tony';
User is a single table. Course is divided into tables and not divided into libraries
### Example codes for reproduce this issue (such as a github link).

--- a/shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/context/OptimizeContextFactory.java
+++ b/shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/context/OptimizeContextFactory.java
@@ -71,6 +71,8 @@ public final class OptimizeContextFactory {
     
     private static final String FUN_CAMEL_NAME = CalciteConnectionProperty.FUN.camelName();
     
+    private static final String TIME_ZONE = CalciteConnectionProperty.TIME_ZONE.camelName();
+    
     @Getter
     private final DatabaseType databaseType;
     
@@ -109,6 +111,7 @@ public final class OptimizeContextFactory {
     
     private void initProperties(final DatabaseType databaseType) {
         // TODO Logic could be improved.
+        properties.setProperty(TIME_ZONE, "UST");
         if (databaseType instanceof MySQLDatabaseType || databaseType == null) {
             properties.setProperty(LEX_CAMEL_NAME, Lex.MYSQL.name());
             properties.setProperty(CONFORMANCE_CAMEL_NAME, SqlConformanceEnum.MYSQL_5.name());

--- a/shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/context/OptimizeContextFactory.java
+++ b/shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/context/OptimizeContextFactory.java
@@ -111,7 +111,7 @@ public final class OptimizeContextFactory {
     
     private void initProperties(final DatabaseType databaseType) {
         // TODO Logic could be improved.
-        properties.setProperty(TIME_ZONE, "UST");
+        properties.setProperty(TIME_ZONE, "UTC");
         if (databaseType instanceof MySQLDatabaseType || databaseType == null) {
             properties.setProperty(LEX_CAMEL_NAME, Lex.MYSQL.name());
             properties.setProperty(CONFORMANCE_CAMEL_NAME, SqlConformanceEnum.MYSQL_5.name());

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/it/BaseITCase.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/it/BaseITCase.java
@@ -42,7 +42,6 @@ import java.sql.SQLException;
 import java.sql.Statement;
 import java.text.ParseException;
 import java.util.Map;
-import java.util.TimeZone;
 
 @Getter(AccessLevel.PROTECTED)
 @RunWith(ShardingSphereRunner.class)
@@ -72,10 +71,6 @@ public abstract class BaseITCase {
     private DataSource targetDataSource;
 
     private DataSource dataSourceForReader;
-    
-    static {
-        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
-    }
     
     BaseITCase(final ParameterizedArray parameterizedArray) {
         this.adapter = parameterizedArray.getAdapter();

